launch/navigation.launch.py

```diff
@@ -40,6 +41,20 @@ def generate_launch_description():
         ]
     )
 
+    # TODO: disable when navigation2 supports twist stamped
+    # Twist stamper.
+    twist_stamper = Node(
+        package="twist_stamper",
+        executable="twist_stamper",
+        parameters=[
+            {"frame_id": "base_link"},
+        ],
+        remappings=[
+            ("cmd_vel_in", "cmd_vel"),
+            ("cmd_vel_out", "ap/cmd_vel"),
+        ],
+    )
```

Comment:
I see, that's a clever and organized way to work around the `nav2` twist situation, good job @srmainwaring!